### PR TITLE
Fix/pair ids

### PR DESCRIPTION
**Pull request by dvilelaf**
## Proposed changes

[Fix the APY pair_ids](https://github.com/valory-xyz/open-autonomy/pull/1171#discussion_r940922042)

## Fixes

n/a

## Types of changes

What types of changes does your code introduce? (A **breaking change** is a fix or feature that would cause existing functionality and APIs to not work as expected.)
_Put an `x` in the box that applies_

- [x] Non-breaking fix (non-breaking change which fixes an issue)
- [ ] Breaking fix (breaking change which fixes an issue)
- [ ] Non-breaking feature (non-breaking change which adds functionality)
- [ ] Breaking feature (breaking change which adds functionality)
- [ ] Refactor (non-breaking change which changes implementation)
- [ ] Messy (mixture of the above - requires explanation!)

## Checklist

_Put an `x` in the boxes that apply._

- [x] I have read the [CONTRIBUTING](../blob/main/CONTRIBUTING.md) doc
- [x] I am making a pull request against the `main` branch (left side). Also you should start your branch off our `main`.
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have locally run services that could be impacted and they do not present failures derived from my changes

### CODE REVIEW

File: packages/valory/skills/apy_estimation_chained_abci/skill.yaml [hunk 1]
```diff
@@ -112,11 +113,11 @@ models:
...
       pair_ids:
-        uniswap_subgraph:
+      - - uniswap_subgraph
         - '0xae461ca67b15dc8dc81ce7615e0320da1a9ab8d5'
-        spooky_subgraph:
-        - '0x2a651563c9d3af67ae0388a5c8f89b867038089e'
-        - '0x2b4c76d0dc16be1c31d4c1dc53bf9b45987fc75c'
+      - - spooky_subgraph
+        - - '0x2a651563c9d3af67ae0388a5c8f89b867038089e'
+          - '0x2b4c76d0dc16be1c31d4c1dc53bf9b45987fc75c'
```
Comment (dvilelaf) [inline packages/valory/skills/apy_estimation_chained_abci/skill.yaml:120]: @angrybayblade following your recommendation, when defining pair_ids as:
`pair_ids: [["uniswap_subgraph", '0xae461ca67b15dc8dc81ce7615e0320da1a9ab8d5'],["spooky_subgraph",['0x2a651563c9d3af67ae0388a5c8f89b867038089e', '0x2b4c76d0dc16be1c31d4c1dc53bf9b45987fc75c']]]`

It gets re-formatted into this form when running make hashes.

Comment (angrybayblade) [reply packages/valory/skills/apy_estimation_chained_abci/skill.yaml:120]: Yeah, it's the yaml format for 2d lists

Comment (angrybayblade) [reply packages/valory/skills/apy_estimation_chained_abci/skill.yaml:120]: Why does `uniswap_subgraph` has only one  address whereas `spooky_subgraph` has two? @dvilelaf any idea?

Comment (dvilelaf) [reply packages/valory/skills/apy_estimation_chained_abci/skill.yaml:120]: No, I don't really know

File: packages/valory/services/apy_estimation/service.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+number_of_agents: 4
+---
+public_id: valory/apy_estimation_abci:0.1.0
+type: skill
+models:
+  params:
+    args:
+      pair_ids:
+        uniswap_subgraph:
+        - '0xae461ca67b15dc8dc81ce7615e0320da1a9ab8d5'
```
Comment (DavidMinarsch) [inline packages/valory/services/apy_estimation/service.yaml:20]: Surely it needs double nesting here too? @dvilelaf

Comment (dvilelaf) [reply packages/valory/services/apy_estimation/service.yaml:20]: I've just added it and also in another place but it seems it breaks tests.

**Comment by dvilelaf** [comment]
Closed in favor of #1189 1189

**Pull request closed without merging**